lib/broccoli/bundler.js

```diff
@@ -148,8 +149,9 @@ module.exports = class Bundler {
     options = options || {};
 
     let scriptOutputFiles = options.scriptOutputFiles;
+    let applicationAndDepsDebugTree = new BroccoliDebug(applicationAndDepsTree, 'bundler:application-and-dependencies');
```

Comment:
Also, I think we should call this `bundle-js:input` (not `'application-and-dependencies'`)